statistics.csv

```diff
@@ -0,0 +1,145 @@
+|*field*|*frequency*| facultative: different values sorted by occurrence , separated with commata*|
```

Comment:
Uh - this file wasn't supposed to be commited, yes?

Reply:
hm, no.

Reply:
I'll remove it